src/core/parseutil.cpp

```diff
@@ -77,7 +77,7 @@ QList<HealLocation>* ParseUtil::parseHealLocs(QString text) {
     int i = 1;
     for (auto line : lines){
         if (line.contains("MAP_GROUP")){
-            QList<QString> li = line.replace(" ","").chopped(2).remove('{').split(',');
```

Comment:
`chopped()` and `left()` do different things, so I doubt this code still behaves correctly.

Reply:
I'd be surprised if the app even opened

Reply:
Ah, I read the documentation wrong. I'll fix this up.

Reply:
this line should work if it's really an issue `QList<QString> li = line.remove(QRegExp("[\{|\}]")).remove(" ").split(",");`

Reply:
I changed it to just subtract 2 from the length like `chopped` does.

Reply:
honestly the line I wrote is better because as it was before, it requires the last item in the array have a trailing comma

Reply:
Okay I changed it to that :ok_hand:

Reply:
```c++
src/core/parseutil.cpp:80:53: warning: unknown escape sequence: '\}'
             QList<QString> li = line.remove(QRegExp("[\{|\}]")).remove(" ").split(",");
```
Should this be `\\{`?

Reply:
no but you can just remove the slash altogether

Reply:
I merged the `remove` calls it and works :ok_hand: